index.html

```diff
@@ -215,10 +215,12 @@ <h1>
   return <ul>{listItems}</ul>;
 }
 
+const Quiz = <QuizContent data={data} />;
+
 ReactDOM.render(
-  <Quiz data={data} />,
+  Quiz,

```

Comment:
Ta zmiana jest mocno zbędna. Stworzyłeś takie dodatkowe coś niepotrzebne pośrednie. Proszę to uporządkować. Poza tym w porząsiu.